cilium/cmd/bpf_lb_list.go

```diff
@@ -108,6 +108,12 @@ var bpfLBListCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		common.RequireRootPrivilege("cilium bpf lb list")
 
+		// Ensure that the BPF map objects have been initialized before trying
+		// to list them. Note, this is _not_ creating a new map, but rather
+		// initializing the Go object representing the map.
+		lbmap.InitializeIPv4()
+		lbmap.InitializeIPv6()
```

Comment:
Couldn't we call this from the functions below with e.g. a `sync.Once`? Seems like it would be less fragile (less likely to call `dumpX` without initializing the map objects).

Reply:
Are you suggesting that we wrap every call to initialize the map with a `sync.Once`? Clearly that would give us assurance that the maps would be initialized once, but on the other hand, someone would have to _explicitly_ call these initialize functions. I can see a `sync.Once` being helpful where the nature of the code is in some sort of loop or the code paths aren't clear, etc.

Reply:
Hm, I'm just seeing this all lives in the same package, so my previous comment probably doesn't make much sense. Sorry for the noise.